src/topp/CometAdapter.cpp

```diff
@@ -533,6 +570,13 @@ class TOPPCometAdapter :
     os << "output_suffix =\n";                                   // add a suffix to output base names i.e. suffix "-C" generates base-C.pep.xml from base.mzXML input
     os << "mass_offsets = " << ListUtils::concatenate(getDoubleList_("mass_offsets"), " ") << "\n"; // one or more mass offsets to search (values subtracted from deconvoluted precursor mass)
     os << "precursor_NL_ions =\n"; //  one or more precursor neutral loss masses, will be added to xcorr analysis 
+    os << "pinfile_protein_delimiter = " << getStringOption_("pinfile_protein_delimiter") << "\n";
+    for (char residue = 'A'; residue <= 'Z'; ++residue) {
```

Comment:
does this make sense for the whole alphabet? Does comet support the whole alphabet (e.g. Z,B,J are not an amino acid but sometimes used for more than one ambiguous amino acids)

Reply:
Comet does it like that https://uwpr.github.io/Comet/parameters/parameters_202402/set_X_residue.html
even though its not entirely clear what it means for the ambiguous AA's.